test/service_objects/game_piece_action_resolvers/melee_attack_resolver.test.ts

```diff
@@ -49,8 +49,7 @@ describe('validateMeleeAttackAction', () => {
       let result = await validateMeleeAttackAction(
         attackingGamePiece,
         targetGamePiece.id,
-        false,
-        null
+        false
```

Comment:
rm `null` from the transaction argument - no longer needed thanks to `transaction?`